jabgui/src/main/java/org/jabref/gui/maintable/ColumnPreferences.java

```diff
@@ -19,6 +19,37 @@ public ColumnPreferences(List<MainTableColumnModel> columns,
         this.columnSortOrder = FXCollections.observableArrayList(columnSortOrder);
     }
 
+    // Private default constructor with hardcoded default values
```

Comment:
Remove this comment - this is not done in the example - and in the other PRs. (Alternatively, use the wording presented at https://github.com/JabRef/jabref/issues/14400)